modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1483,7 +1483,12 @@ public void testAddObjectEntryWithObjectValidationRule() throws Exception {
 			).put(
 				"listTypeEntryKeyRequired", "listTypeEntryKey1"
 			).put(
-				"time", dateTimeFormatter.format(LocalDateTime.now())
+				"time",
+				dateTimeFormatter.format(
```

Comment:
Aiming improve slight the readability and also save some lines, you can create a variable to this dateTime string and apply where needed.

```java 
String tomorrowLocalDateTime = dateTimeFormatter.format(
					LocalDateTime.now(
					).plusDays(
						1
					));
```